lib/pdn/pdn.c

```diff
@@ -601,5 +639,11 @@ static void on_cfun(enum lte_lc_func_mode mode, void *ctx)
 			LOG_ERR("Unable to subscribe to +CGEREP=1, err %d", err);
 		}
 	}
+
+	if (mode == LTE_LC_FUNC_MODE_POWER_OFF) {
+#if defined(CONFIG_PDN_DEFAULTS_OVERRIDE)
+		pdn_defaults_override();
+#endif
```

Comment:
This is a best-effort solution, i.e. if app does not use lib_lte_lc at all, it could still configure the default PDN by calling lib_pdn API.